src/Umbraco.Web.BackOffice/Filters/SendingContentNotification.cs

```diff
@@ -0,0 +1,19 @@
+using Umbraco.Cms.Core.Events;
+using Umbraco.Cms.Core.Models.ContentEditing;
+using Umbraco.Cms.Core.Web;
+
+namespace Umbraco.Cms.Web.BackOffice.Filters
+{
+    public class SendingContentNotification : INotification
```

Comment:
I feel like it makes sense to make all `INotification` implementing classes as `sealed` since I don't really see a scenario where they would be inherited? What do you think? cc @bergmania ?

Reply:
Cool with me..

Reply:
Makes sense. I've got a huge PR incoming for content and media events rewrite... I'll make all the new notifications there sealed, then I can revisit the already merged ones afterwards.